app/js/thisissoon-nav/navCounter.js

```diff
@@ -29,17 +29,27 @@ angular.module("thisissoon.nav.snNavCounter", [
                 $scope.sections = NAV;
                 $scope.sectionCurrent = 1;
 
-                var activeNav, activeLi, activeIndex, ul;
+                var activeLi, activeIndex, ul;
 
```

Comment:
It's always good practice to write documentation for all your variables and functions so you can refer to it when you come back to it in the future if you forget and more importantly for others looking at your code